### PR TITLE
rental controller

### PR DESCRIPTION
**Pull request by maksigors**

### CODE REVIEW

**Review by mykyta-udav — APPROVED** [review]

File: src/main/java/mate/academy/car/sharing/controller/RentalController.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    @GetMapping
+    public List<RentalResponseDto> getAll() {
+        return rentalService.getAll().stream()
+                .map(mapper::mapToDto)
+                .collect(Collectors.toList());
+    }
+
+    @Operation(summary = "Get all rentals by user ID", description = "Get all rentals by user ID")
+    @GetMapping("/user")
+    public List<RentalResponseDto> getByUserId(@RequestParam Long userId,
```
Comment (Tupadzaky) [inline src/main/java/mate/academy/car/sharing/controller/RentalController.java:35]: this method is the same as getAll(). Fix it please, so it actually uses userId and isActive

File: src/main/java/mate/academy/car/sharing/entity/Rental.java [hunk 2]
```diff
@@ -31,3 +31,7 @@ public class Rental {
     @JoinColumn(name = "user_id")
     private User user;
 }
+
+
+
+
```
Comment (Tupadzaky) [inline src/main/java/mate/academy/car/sharing/entity/Rental.java:37]:
```suggestion
```

File: src/main/java/mate/academy/car/sharing/service/RentalService.java [hunk 3]
```diff
@@ -1,6 +1,8 @@
 package mate.academy.car.sharing.service;
 
+import java.util.List;
 import mate.academy.car.sharing.entity.Rental;
 
 public interface RentalService extends AbstractService<Rental> {
+    List<Rental> getByUserIdActual(Long userId);
```
Comment (Tupadzaky) [inline src/main/java/mate/academy/car/sharing/service/RentalService.java:7]: This method is not used. Fix this please

File: src/main/java/mate/academy/car/sharing/service/impl/RentalServiceImpl.java [hunk 4]
```diff
@@ -22,6 +22,11 @@ public Rental getById(Long id) {
         return rentalRepository.getReferenceById(id);
     }
 
+    @Override
+    public List<Rental> getByUserIdActual(Long userId) {
+        return List.of();
```
Comment (Tupadzaky) [inline src/main/java/mate/academy/car/sharing/service/impl/RentalServiceImpl.java:27]: Fix this please

**Review by Tupadzaky — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/car/sharing/controller/RentalController.java (hunk 1 above)
Comment (maksigors) [reply src/main/java/mate/academy/car/sharing/controller/RentalController.java:35]: it's temperally

**Pull request closed without merging**